cmd/jimmsrv/service/service.go

```diff
@@ -398,6 +399,12 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 		JWTService:                 s.jimm.JWTService,
 	}
 
+	roleManager, err := role.NewRoleManager(&s.jimm.Database, s.jimm.OpenFGAClient)
+	if err != nil {
+		return nil, errors.E(op, err, "failed to create RoleManager")
+	}
+	s.jimm.RoleManager = roleManager
+
```

Comment:
i don't know if I am sold on this.

This is inconsistent with the common pattern we've used all around the code. It's a start going towards breaking up JIMM, but I would've preferred this in a separate PR

Reply:
We have jwkservice, oauthauthenticator, etc. They're all the same pattern.

Reply:
I agree that it's unfortunate it's so inconsistent with the rest of the business logic. I think Simone is also thinking about the other jimm methods that form all the business logic and how this is very different from that.
Though of course we can get those to look more like this going forward.

Reply:
i would love to split up the jimm package into smaller services that are easier to test